docs/develop/configure-your-add-in-to-use-a-shared-runtime.md

```diff
@@ -55,7 +55,8 @@ Follow these steps for a new or existing project to configure it to use a shared
 1. Find the **\<VersionOverrides\>** section and add the following **\<Runtimes\>** section. The lifetime needs to be **long** so that your add-in code can run even when the task pane is closed. The `resid` value is **Taskpane.Url**, which references the **taskpane.html** file location specified in the `<bt:Urls>` section near the bottom of the **manifest.xml** file.
 
     > [!IMPORTANT]
-    > The **\<Runtimes\>** section must be entered after the **\<Host\>** element in the exact order shown in the following XML.
+    > The shared runtime won't load if the `resid` uses different values in the manifest. If you change the value to something other than **Taskpane.Url** be sure to also change the value in all locations shown in the following steps in this article.
+    > Also, the **\<Runtimes\>** section must be entered after the **\<Host\>** element in the exact order shown in the following XML.
```

Comment:
Is the intention that this should all be in one paragraph? If not, then consider adding a newline
```suggestion
    >
    > Also, the **\<Runtimes\>** section must be entered after the **\<Host\>** element in the exact order shown in the following XML.
```